cli/src/commands/describe.rs

```diff
@@ -68,6 +68,12 @@ pub(crate) struct DescribeArgs {
     /// This is mainly useful in combination with e.g. `--reset-author`.
     #[arg(long)]
     no_edit: bool,
+    /// Open an editor
+    ///
+    /// This is mainly useful to reverse the effects of --no-edit or another
+    /// flag/option that implies --no-edit e.g. --message
+    #[arg(long)]
```

Comment:
suggestion: Should this conflict with `--no-edit`? (See e.g. https://docs.rs/clap/latest/clap/struct.Arg.html#method.conflicts_with, or there's probably examples in other places we have conflicting arguments).

suggestion (non-blocking): Hmm, it seems like `--message` and `--stdin` should also be marked as conflicting, since one is silently ignored if the other is passed, so if you have free time, that could be a good change to make as well 🙂.

Reply:
1. I thought about that, but decided against it because it's not uncommon to have `--x`/`--no-x` flags that can both be used and can override each other.  I also implemented the change such that `--edit` explicitly overrides `--no-edit` (regardless of order), and added a test to that effect.  I could be convinced otherwise though!
2. That's a good idea, I'll add that.  I should add that in a separate commit in this PR, right?

Reply:
--stdin and --message are now marked as conflicting.  I'll resolve this given that the rest of the comment is being discussed elsewhere.